src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2115,42 +2240,56 @@
           short-summary: Port number on which to receive traffic.
         - name: --dest-address
           short-summary: The IP address or URI at which to receive traffic.
+    examples:
+        - name: This example checks connectivity to a destination virtual machine over port 80.
+          text: az network watcher test-connectivity -g ContosoRG --source-resource MultiTierApp0 --dest-resource Database0 --dest-port 80
 """
 
 helps['network watcher show-next-hop'] = """
     type: command
     short-summary: Get information on the 'next hop' for a VM.
-    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located.
+    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located.  For more information about show-next-hop, refer to https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-check-next-hop-cli
+    examples:
+        - name: Use show-next-hop to get the next hop from a VM to a given destination IP address
```

Comment:
Do not use the command name in examples. Focus on the task being performed. Suggestion:

```
Get the next hop from a VM's assigned IP address to a destination at 10.1.0.4.
```

This also allows for an improvement: Getting the 1st assigned VNET IP address from a VM, and then using that as the source.

Reply:
Addressed.